src/connectors/breadcrumb/connectBreadcrumb.ts

```diff
@@ -73,7 +78,8 @@ export type BreadcrumbRendererOptions = {
 
 export type BreadcrumbConnector = Connector<
   BreadcrumbRendererOptions,
-  BreadcrumbConnectorParams
+  BreadcrumbConnectorParams,
+  BreadcrumbWidgetRenderState
```

Comment:
I thought we were going to move calculate RenderState in the Connector type directly, so the arguments could stay as-is?

Reply:
Oh sorry. I forgot to mention this.

I think we can focus on the typing around the widget itself and `connectAutocomplete` as an example.
Once it's all agreed and this PR is merged, then we can fix types in new PRs, one per connector.

Reply:
https://github.com/algolia/instantsearch.js/pull/4518/commits/d7e5206a393a3af6164e57d3691139f26acb1dbd